webapp/utils/aws/ecs.py

```diff
@@ -0,0 +1,90 @@
+import logging
+import re
+import time
+from typing import TYPE_CHECKING
+
+import boto3
+from attrs import define
+
+if TYPE_CHECKING:
+    from mypy_boto3_ecs.client import ECSClient as ECSClientType
+
+from webapp.exceptions import ECSTaskDefinitionDoesNotExistError
+
+logger = logging.getLogger(__name__)
+
+
+@define
+class ECSClient:
+    """ECS Client for running and monitoring task runs."""
+
+    cluster: str
+    task_definition: str
+    network_configuration: dict
+
+    @property
+    def client(self) -> "ECSClientType":
+        return boto3.client("ecs")
+
+    @property
+    def container_name(self) -> str:
+        return self.cluster.split("/")[-1]
+
+    @property
+    def task_family(self) -> str:
+        task_family_revision = self.task_definition.split("/")[-1]
+        return re.sub(":.*", "", task_family_revision)
+
+    def execute_review_run(self) -> dict | None:
+        if self.task_exists():
+            response = self.client.run_task(
+                cluster=self.cluster,
+                launchType="FARGATE",
+                networkConfiguration=self.network_configuration,  # type: ignore[arg-type]
+                overrides={
+                    "containerOverrides": [
+                        {
+                            "name": self.container_name,
+                            "command": ["--blahblah"],  # [TODO] Update for SAP Invoices
+                        }
+                    ]
+                },
+                taskDefinition=self.task_definition,
+            )
+            return self.parse_run_details(response)  # type: ignore[arg-type]
```

Comment:
For now, I've only written a `execute_review_run` method. Later, an `execute_final_run` method will be implemented as well. I am hoping that we can sort out some orchestration + output discussions from a review run first!